packages/lib/ren-tx/src/machines/deposit.ts

```diff
@@ -450,11 +450,15 @@ export const depositMachine = Machine<
         guards: {
             isSrcSettling: ({
                 deposit: { sourceTxConfs, sourceTxConfTarget },
-            }) => (sourceTxConfs || 0) < (sourceTxConfTarget || 1),
+            }) =>
+                (sourceTxConfs || 0) <
+                (sourceTxConfTarget || Number.POSITIVE_INFINITY), // If we don't know the target, keep settling
```

Comment:
If `sourceTxConfTarget` is 0 (e.g. for some testnet assets), does this still work as intended?

Reply:
This should be ok; it will only result in entering an early state, and check for confirmations when it doesn't really need to (but ren-js will resolve the mint and proceed)